mam/v2/ntru/tests/test_ntru.c

```diff
@@ -15,44 +15,14 @@
 #include "mam/v2/ntru/poly.h"
 #include "mam/v2/test_utils/test_utils.h"
 
-static void ntru_test_gen(mam_ntru_sk_t *n, trits_t rf, trits_t rg,
-                          trits_t pk) {
-  poly_coeff_t *f;
-  MAM2_POLY_DEF(g);
-  MAM2_POLY_DEF(h);
-
-  f = (poly_coeff_t *)n->f;
-  poly_small_from_trits(f, rf);
-  poly_small_from_trits(g, rg);
-
-  /* f := NTT(1+3f) */
-  poly_small_mul3(f, f);
```

Comment:
Could you modify the test instead of deleting it?

Reply:
That's actually an unused function that was used by the decryption failure that we removed in a previous PR